selfservice/hook/verification.go

```diff
@@ -147,23 +151,20 @@ func (e *Verifier) do(
 		}
 
 		verificationFlow.State = flow.StateEmailSent
-
 		if err := strategy.PopulateVerificationMethod(r, verificationFlow); err != nil {
 			return err
 		}
 
-		if address.Value != "" && address.Via == identity.VerifiableAddressTypeEmail {
-			verificationFlow.UI.Nodes.Append(
-				node.NewInputField(address.Via, address.Value, node.CodeGroup, node.InputAttributeTypeSubmit).
-					WithMetaLabel(text.NewInfoNodeResendOTP()),
-			)
-		}
+		verificationFlow.UI.Nodes.Append(
```

Comment:
I think this will fix inconsistencies when using SMS channel. The resend button was probably removed to solve issues around resending the flow, but it's anyways not very effective in my view and should instead be solved with rate limiting or similar